frontend/src/graphql/authUser.graphql

```diff
@@ -1,6 +1,5 @@
 mutation AuthUser($token: String!, $email: String!, $name: String!) {
-  authUser(id: $id, token: $token, email: $email, name: $name) {
-    id
+  authUser(token: $token, email: $email, name: $name) {
     email
```

Comment:
It is a good practice to always return the ID property in graphql queries
It helps apollo cache objects that are returned in your graphql requests

You can read more about how apollo does caching here:
https://www.apollographql.com/docs/react/caching/overview/

Reply:
Done!